### PR TITLE
docstring changes for memmodel/preprocessing

### PR DESCRIPTION
**Pull request by abhsharma2**

### CODE REVIEW

File: verticapy/machine_learning/memmodel/preprocessing.py [hunk 1]
```diff
@@ -125,14 +184,74 @@ def __init__(self, mean: ArrayLike, std: ArrayLike) -> None:
...
+
+    min\_: ArrayLike
         Model's features minimums.
-    max_: ArrayLike
+    max\_: ArrayLike
         Model's features maximums.
+
+    .. note:: :py:mod:`verticapy.machine_learning.memmodel` are defined
+        entirely by their attributes. For example, 'minimum',
+        and 'maximum' values of feature(S) defines a MinMaxScaler model.
```
Comment (oualib) [inline verticapy/machine_learning/memmodel/preprocessing.py:199]: please change the sentence:
... 'maximum' values of the different features define a ...

File: verticapy/machine_learning/memmodel/preprocessing.py [hunk 2]
```diff
@@ -125,14 +184,74 @@ def __init__(self, mean: ArrayLike, std: ArrayLike) -> None:
...
+
+    **Initalization**
+
+    Import the required module.
+
+    .. ipython:: python
+
+        from verticapy.machine_learning.memmodel.preprocessing import MinMaxScaler
+
+    A MinMaxScaler model is defined by minimum and maximum values. In this example, we will use the following:
```
Comment (oualib) [inline verticapy/machine_learning/memmodel/preprocessing.py:212]: sentence is too long, please try to go back to line when it is the case.

Comment (oualib) [reply verticapy/machine_learning/memmodel/preprocessing.py:212]: same below

**Review by oualib — APPROVED** [review]
approve, I'll quickly resolve the small issues.

File: verticapy/machine_learning/memmodel/preprocessing.py [hunk 3]
```diff
@@ -150,26 +296,103 @@ def __init__(self, min_: ArrayLike, max_: ArrayLike) -> None:
...
+        drop_first = False
+        column_naming = None
+
+    Let's create a
+    :py:mod:`verticapy.machine_learning.memmodel.preprocessing.OneHotEncoder`
+    model.
+
+    .. ipython:: python
+
+        model_ohe = OneHotEncoder(categories, drop_first, column_naming)
```
Comment (mail4umar) [inline verticapy/machine_learning/memmodel/preprocessing.py:358]: for consistency and ease of users to copy, can you combine it into one block i.e

``` 
model = OneHotEncoder(
        categories = categories,
        drop_first = False,
        column_naming = "values"
)

```

**Review by mail4umar — CHANGES_REQUESTED** [review]

File: verticapy/machine_learning/memmodel/preprocessing.py [hunk 4]
```diff
@@ -150,26 +296,103 @@ def __init__(self, min_: ArrayLike, max_: ArrayLike) -> None:
...
+
+    .. ipython:: python
+
+        from verticapy.machine_learning.memmodel.preprocessing import OneHotEncoder
+
+    A OneHotEncoder model is defined by categories, column naming
+    criteria and drop_first flag. In this example, we will use the
+    following:
+
+    .. ipython:: python
```
[the remark this answers is not in the document]
Comment (oualib) [reply verticapy/machine_learning/memmodel/preprocessing.py:346]: it would be hard for this example

**Review by mail4umar — APPROVED** [review]

**Comment by mail4umar** [comment]
> @mail4umar everything is updated, try to look at the render on your end.

all looks good! you can merge when tests pass.

**Pull request merged**